hls4ml/templates/vivado/nnet_utils/nnet_code_gen.h

```diff
@@ -25,6 +25,96 @@ template <class data_T, typename CONFIG_T> class FillConv2DBuffer {
     }
 };
 
+template <class data_T, class res_T, typename CONFIG_T> class PointwiseConv1D {
+  public:
+    static void pointwise_conv(data_T data[CONFIG_T::in_width * CONFIG_T::n_chan],
+                               res_T res[CONFIG_T::out_width * CONFIG_T::n_filt],
+                               typename CONFIG_T::weight_t weights[CONFIG_T::n_chan * CONFIG_T::n_filt],
+                               typename CONFIG_T::bias_t biases[CONFIG_T::n_filt]) {
+        // To be implemented in subclasses
+    }
+};
+
+template <class data_T, class res_T, typename CONFIG_T>
+void pointwise_conv_1d_latency_cl(data_T data[CONFIG_T::in_width * CONFIG_T::n_chan / CONFIG_T::reuse_factor],
```

Comment:
Why is this here?

Reply:
I was worried about some circular imports, but actually it seems ok... I moved it back to `nnet_cov1d_latency.h`